src/main/java/frc/robot/RobotContainer.java

```diff
@@ -35,31 +35,32 @@
  * commands, and button mappings) should be declared here.
  */
 public class RobotContainer {
-  private Joystick m_driveJoystick;
-  private Joystick m_operatorJoystick;
-
-  private DriveTrain m_driveTrain;
-  private Shooter m_shooter;
-  private Climber m_climber;
-  private Limelight m_limelight;
-  private PowerDistributionPanel m_pdp;
-  private Revolver m_revolver;
-  private Intake m_intake;
-  private Turret m_turret;
+  private final Joystick m_driveJoystick;
+  private final Joystick m_operatorJoystick;
+  private final DriveTrain m_driveTrain;
+  private final Shooter m_shooter;
```

Comment:
Can you please re-add the blank line between joysticks and the subsystems? Since they're different categories of objects, I think it makes sense to separate them. Same below in the constructor.

Reply:
Fixed in commit f0ee4bb